src/main/java/Juan.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+
+public class Juan {
+    public static void main(String[] args) {
+        lineMessage();
+        helloMessage();
+        lineMessage();
+
+        boolean continueChatting = true;
+        while (continueChatting) {
+            continueChatting = chatFeature();
+        }
+        byeMessage();
+        lineMessage();
+    }
+    public static boolean chatFeature(){
+        // Less efficient to create a new scanner everytime but code is much neater
+        // If return True means continue
+        // Else End
+
+        Scanner scanner = new Scanner(System.in);
+        String line = scanner.nextLine();
+        lineMessage();
+
+        if (line.equals("bye")) {
+            return false;
+        } else if (line.equals("list")) {
+            Task.printTasksList();
+            return true;
+        }
+
+        // Check for mark and unmark or just add task
+        String[] parts = line.split(" ");
+        if (parts[0].equals("mark")){
+            // Mark
+            int taskIndex = Integer.parseInt(parts[1]) - 1;
+            Task.mark(taskIndex);
+        } else if (parts[0].equals("unmark")){
+            // Unmark
+            int taskIndex = Integer.parseInt(parts[1]) - 1;
+            Task.unmark(taskIndex);
+        } else {
+            // else add task
+            Task newTask = new Task(line);
+        }
+
+        lineMessage();
+        return true;
+    }
+
+    // Message Functions for cleaner main Function
+    public static void lineMessage() {
+        String line = "____________________________________________________________\n";
+        System.out.print(line);
+    }
```

Comment:
For the messages, should it be static variables instead and what the methods do is just printing it out?